### ISSUE TITLE
required ruby version

### ISSUE THREAD

**Issue by DanielBoettner** [centosadmin/redmine_recurring_tasks#13]
Hello,

is there any information if the plugin works with ruby 2.1.9?
I have seen the requirements but only after installing the plugin.

Would love to know if a ruby upgrade is required.
Cause the plugin already appears in the admin section.

But apart from enabling the module in a project I do see no options / fields / ...
that would allow to create a recurring task.

Any help is appreciated.

**Comment by constXife** [comment]
Hi, Daniel. Thank you for your question.

Plugin should work with ruby 2.1, as I see. All tests are passed successfully. 

To make schedule for any redmine issue you should open an issue, then you will see in details box "Schedule" and you should click "Add" link.

**Comment by DanielBoettner** [comment]
Thanks for the quick response.

I struggled to see it. Tried to find it while creating a new issue or while editing an issue.
Very useful plugin. Keep up the good work.
